Release/CMakeLists.txt

```diff
@@ -164,7 +165,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR IOS)
 elseif(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
   message("-- Setting gcc options")
 
-  set(WARNINGS -Wall -Wextra -Wunused-parameter -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-format-attribute -Wmissing-include-dirs -Wpacked -Wredundant-decls -Wunreachable-code)
+  set(WARNINGS -Wall -Wextra -Wunused-parameter -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-format-attribute -Wno-unused-parameter -Wmissing-include-dirs -Wpacked -Wredundant-decls -Wunreachable-code)
```

Comment:
`-Wunused-parameter -Wno-unused-parameter` seems... counterintuitive?

Reply:
Keeping  -Wno-unused-parameter and removing  -Wunused-parameter